docs/benchmarks/profiling.md

```diff
@@ -0,0 +1,122 @@
+---
+sidebar_position: 10
+---
+
+# Profiling Hydra scripts
+
+This is a quick tutorial how to profile Hydra scripts and is intended for
```

Comment:
typos: 
* `a quick tutorial` -> `a quick tutorial on`
* `and is intended` -> `, and it is intended`